zulip/zulip/__init__.py

```diff
@@ -867,6 +867,19 @@ def upload_custom_emoji(self, emoji_name: str, file_obj: IO[Any]) -> Dict[str, A
             files=[file_obj]
         )
 
+    def update_realm(self, **request: Any) -> Dict[str, Any]:
+        '''
+            Example usage:
+
+            >>> client.update_realm({'digest_weekday': 0', 'signup_notifications_stream_id': -1})
```

Comment:
The quoting here seems likely hand-written and wrong.  E.g. the `0` with a quote on only one side of it.

Please be sure to test your examples!

Reply:
oops, sorry! I write it by hand indeed, actually, I copy/paste the example of the `/realm` documentation on `zulip.yaml`. I'll do an actual request and copy the result here.

Reply:
I guess this still isn't resolved.